LocalSettings.php

```diff
@@ -1704,6 +1704,9 @@
 	'wgInterwikiCentralDB' => array(
 		'default' => 'metawiki',
 	),
+	'wgExtraInterlanguageLinkPrefixes' => array(
+		'tallercentralwiki' => voy,
```

Comment:
The word “voy” needs to be enclosed with an apostrophe on both sides

Reply:
uh, anything else?